app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -362,15 +362,13 @@ private void launchZoomActivityAfterPermissionCheck(final View view) {
     @Override
     public void onResume() {
         super.onResume();
-        if (getParentFragment() != null && getParentFragment().getParentFragment() != null) {
-            //Added a check because, not necessarily, the parent fragment will have a parent fragment, say
-            // in the case when MediaDetailPagerFragment is directly started by the CategoryImagesActivity
-            if (getParentFragment() instanceof ContributionsFragment) {
-                ((ContributionsFragment) (getParentFragment()
-                    .getParentFragment())).binding.cardViewNearby
-                    .setVisibility(View.GONE);
-            }
+
+        //Hide the Nearby card when looking at media details
+        ContributionsFragment cf = this.getContributionsFragmentParent();
```

Comment:
In Java we prefer explicit variable names, even if long. So unless there is a reason I missed, would you mind renaming `cf` to `contributionFragment`?